Libraries/LibGfx/Color.h

```diff
@@ -491,6 +491,7 @@ class Color {
     ByteString to_byte_string() const;
     ByteString to_byte_string_without_alpha() const;
     static Optional<Color> from_string(StringView);
+    static Optional<Color> from_utf16_string(Utf16View);
```

Comment:
`Utf16View` (like `Utf8View`) should be passed by `const&`, it's a bit big to pass by value.